base/ipv4-deployment.yaml

```diff
@@ -22,6 +22,10 @@ spec:
       containers:
         - name: ipv4
           image: public.ecr.aws/cds-snc/notify-ipv4-geolocate-webservice:latest
+          resources:
+            requests:
+              cpu: "50m"
+              memory: "100Mi"
```

Comment:
Just curious where these values come from, are these simply norms?

Reply:
You can read this section to know what are the units on Kubernetes https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-units-in-kubernetes

About the values, I explained in the PR description how I picked those 👍